libs/eavmlib/src/gpio.erl

```diff
@@ -104,6 +86,31 @@ start() ->
 open() ->
     open_port({spawn, "gpio"}, []).
 
+%%-----------------------------------------------------------------------------
+%% @param   GPIO pid that was returned from gpio:start/0
+%% @returns ok atom
+%% @doc     Stop the GPIO interrupt port
+%%
+%%          This function disables any interrupts that are set, stops
+%%          the listening port, and frees all of its resources.
+%% @end
+%%-----------------------------------------------------------------------------
+-spec close(GPIO :: gpio()) -> ok | error.
+close(GPIO) ->
+    port:call(GPIO, {close}).
+
+%%-----------------------------------------------------------------------------
+%% @returns ok atom
+%% @doc     Stop the GPIO interrupt port
+%%
+%%          This function disables any interrupts that are set, stops
+%%          the listening port, and frees all of its resources.
+%% @end
+%%-----------------------------------------------------------------------------
+-spec stop() -> ok | error.
+stop() ->
+    port:call(whereis(gpio), {close}).
```

Comment:
How about `close(whereis(gpio))`?  (just for maintenance reasons)

Reply:
fixed.